src/main/java/com/google/codeu/data/TeamDatastore.java

```diff
@@ -61,4 +64,14 @@ public Team getTeam(Entity teamEntity) {
         String githubLink = (String) teamEntity.getProperty("githubLink");
         return new Team(teamID, cohortID, teamName, projectName, projectDesc, githubLink);
     }
+    
+    public Set<String> getTeams(){
```

Comment:
You'll want to add an input for the teamId:
```suggestion
    public Set<String> getTeams(String teamId){
```